### PR TITLE
Init `just` as command-line runner

### PR DESCRIPTION
**Pull request by Splines**
In this PR, we introduce support for the amazing [`Just` command runner](https://just.systems/) 🔥. This will allow us to specify frequently-used commands and have them available at our fingertips, e.g. `just test cypress`.

## Installation

> [!warning]
> **If you have previously installed a version of `just` on your system via `apt`, please remove it**: `sudo apt remove just`. This is because the version on `apt` is too old and we want to have the latest features of `just` available.
>
> Instead, I've added a script to `package.json`. **Simply run**: `yarn just-install`. It will install `just` to `~/bin`. Then reload your shell and you're good to go. As of today, `just --version` should give `1.29.1`.

Note I've added a VSCode workspace extension recommendation, feel free to install it to get syntax highlighting in the `.justfile`s.

> [!tip]
> `just` modules are currently not stable yet, but will probably be in the next few days (casey [announced](https://github.com/casey/just/issues/929#issuecomment-2138551161) the stabilization to happen on July 1st). For right now, execute `export JUST_UNSTABLE=1` in your shell before executing any `just` command. This PR will only be merged once the `just` modules are stable.

## Usage
Run `just` to see available commands. Run `just test` etc. to see available commands for the submodules. Everything else should be self-explanatory, e.g. try out `just docker up -d`, then `just docker shell db`.

If you want to add your own commands, refer to the [docs](https://just.systems/man/en/). It also has a search built-in.

## TODOs
- [x] Update [Wiki](https://github.com/MaMpf-HD/mampf/wiki/Installation-&-Setup#setting-up-your-workspace-as-a-dev-team-member) on how to install Just.
- [x] Add some more basic commands
- [x] Wait till modules are stabilized, see [this issue](https://github.com/casey/just/issues/929) -> or maybe don't as nothing substantial will change anymore and modules will be stabilized very soon

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
         "shopify.ruby-lsp",
         "dbaeumer.vscode-eslint",
         "streetsidesoftware.code-spell-checker",
-        "streetsidesoftware.code-spell-checker-german"
+        "streetsidesoftware.code-spell-checker-german",
+        "nefrob.vscode-just-syntax"
```
Comment (Splines) [inline .vscode/extensions.json:7]: Feel free to install that recommended extension ;)

**Review by fosterfarrell9 — CHANGES_REQUESTED** [review]
Works nicely. Just one question: Is there a wiki page on just in our wiki? I could not find one, but it would be really helpful.

**Comment by Splines** [comment]
I've added a section about how to install Just [here in the Wiki](https://github.com/MaMpf-HD/mampf/wiki/Installation-&-Setup#setting-up-your-workspace-as-a-dev-team-member). However, you're right it might be better to have a separate section for it for quick reference. I can then reference that section on the installation page.

**Comment by Splines** [comment]
I've outsourced the section to [this new section](https://github.com/MaMpf-HD/mampf/wiki/Installation-&-Setup#installation-of-just-command-line-runner) explaining how to install/update `just`.

**Review by fosterfarrell9 — APPROVED** [review]

**Comment by Splines** [comment]
Note you can now run `yarn just-update` and should get the version `1.33.0`. In this version, modules are stabilized 🙌

**Pull request merged**